src/cache.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+// TODO: move to its own package
+
+import fs from 'fs-extra'
+import moment from 'moment'
+
+type Options = {
+  cacheFn: Function
```

Comment:
rather than this being a generic function, it would be good if you made the type `(Output) => Promise<Array<string>>`